cliff.toml

```diff
@@ -38,9 +63,7 @@ filter_unconventional = true
 # process each line of a commit as an individual commit
 split_commits = false
 # regex for preprocessing the commit messages
-commit_preprocessors = [
-  { pattern = '\((\w+\s)?#([0-9]+)\)', replace = "([#${2}](https://github.com/CQCL/portgraph/issues/${2}))"}, # replace issue numbers
-]
+commit_preprocessors = [{ pattern = '\((\w+\s)?#([0-9]+)\)', replace = "" }]
```

Comment:
What is this trying to achieve? It looks like it's trying to delete things of the form "fixes #123"?

Reply:
The goal is to strip all the prefixes which show the PR number from the commit title.I removed the replacement part since the cliff+github writes that link reading the metadata, which seems more robust than the hardcoded link.

As for how we where generating correct links in the changelog even though here it linked to `portgraph`, I'm not quite sure.
We don't call `git-cliff` directly, but use `release-plz` which does a lot of pre-processing and collects each relevant commit individually before generating the changelog delta programatically. It may be ignoring the `commit_preprocessors` section since it's not using `git-cliff`'s CLI.

This config I submit here produces the correct result locally, but I'll have to review what gets generated by release-plz and submit a fix PR if needed.